Sources/MergeL10n/Commands/PseudoToLanguages.swift

```diff
@@ -102,7 +108,7 @@ struct PseudoToLanguages: ParsableCommand {
     private func run(folder: String, languages: [String], developmentLanguage: String) -> Reader<SimpleFileManager, Result<Void, LocalizedStringFileError>>{
         Reader { fileManager in
             LocalizedStringFile(basePath: folder, language: "zz")
-                .read(encoding: .utf16, requiresComments: true)
+                .read(encoding: .ascii, requiresComments: true)
```

Comment:
Why this change?

Reply:
Thanks for catching. It should not be, I will check it out.

Reply:
```suggestion
                .read(encoding: . utf16, requiresComments: true)
```